ui/src/pages/frontsite/Documents/DocumentsSearch/SeriesListEntry/SeriesListEntry.js

```diff
@@ -21,7 +22,9 @@ export default class SeriesListEntry extends Component {
         as={Link}
         to={FrontSiteRoutes.seriesDetailsFor(this.metadata.pid)}
       >
-        <SeriesImage metadata={this.metadata} />
+        <SeriesImage
```

Comment:
I know that you didn't introduce the, but if you could rename `SeriesImage` to `SeriesCover`, to keep it consistent with the Document. I think it would be better.